.github/workflows/test.yml

```diff
@@ -58,8 +57,7 @@ jobs:
       - name: Install dependencies
         run: |
           python -m pip install --upgrade pip
-          pip install -e .
-          pip install pytest
+          pip install -e .[developer]
           pip install pytest-cov[toml]
           pip install protobuf==3.20.0
 
```

Comment:
Can you move this to a different PR?

Reply:
https://github.com/IntelLabs/MART/pull/59

I guess there's no `pyproject.toml` when he set up CI.